src/main/Config.cpp

```diff
@@ -159,6 +159,7 @@ Config::Config() : NODE_SEED(SecretKey::random())
     CATCHUP_COMPLETE = false;
     CATCHUP_RECENT = 0;
     EXPERIMENTAL_PRECAUTION_DELAY_META = false;
+    EXPERIMENTAL_BACKGROUND_OVERLAY_PROCESSING = true;
```

Comment:
This is temporary for testing, as initially this flag should be `false` by default.